packages/trader/src/sass/app/modules/trading.scss

```diff
@@ -1053,3 +1047,18 @@
         }
     }
 }
+
+.tablet-landscape {
+    position: relative;
+    padding: 0;
+    margin: 0;
+    height: 100%;
```

Comment:
Sorry for silly question, but do we need all three css properties for height?

Reply:
Updated